playwright/e2e/osd-gcp/osd-curated-gcp-wif-psc-cluster-creation.spec.ts

```
@@ -1,12 +1,12 @@
 import { test, expect } from '../../fixtures/pages';
 
-const clusterProperties = require('../../fixtures/osd-gcp/osd-curated-gcp-sa-psc-cluster-creation.spec.json');
+const clusterProperties = require('../../fixtures/osd-gcp/osd-curated-gcp-wif-psc-cluster-creation.spec.json');
```

ESLint

[error] Expected 1 empty line after require statement not followed by another require.

(import/newline-after-import)